runtime/queries/elixir/injections.scm

```diff
@@ -11,6 +11,15 @@
 ((sigil
   (sigil_name) @_sigil_name
   (quoted_content) @injection.content)
- (#match? @_sigil_name "^(h|H)$")
+ (#eq? @_sigil_name "H")
  (#set! injection.language "heex")
  (#set! injection.combined))
+
+(unary_operator
+  operator: "@"
+  operand: (call
+  target: ((identifier) @_identifier (#any-of? @_identifier "moduledoc" "typedoc" "shortdoc" "doc"))
```

Comment:
```suggestion
  target: ((identifier) @_identifier (#match? @_identifier "^(module|type|short)?doc$"))
```

Helix doesn't implement `#any-of?` yet, we've been using `#match?` to get around it

(I think we've discussed it before and the only tricky part of implementing it is that it would most likely be slower than an equivalent `#match?` regex like above to loop through the options. So we might be able to compile it down to a regex for the best of both worlds.)

Reply:
Good to know! I applied the suggestion above; is there anything else you'd like to see in this PR?

Reply:
Nope this looks great! The markdown highlights look really nice, especially the injection in `## Examples` blocks 🤩